curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f6b2d164f81809efd9bdc.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-43
 
 # --description--
 
-After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Also wrap the words `Total Fat 8g` in an additional `span` element for alignment.
+After your last `.divider` element, create a `p` element and give it the text `Total Fat 8g 10%`. Wrap `Total Fat` in a `span` element with the `class` set to `bold`. Wrap `10%` in another `span` element with the `class` set to `bold`. Finally, nest these two `span` elements in an additional `span` element for alignment.
```

Comment:
The desired outcome is:

```html
<p>
  <span>
    <span class="bold">Total Fat</span> 8g
  </span> 
  <span class="bold">10%</span>
</p>
```

These instructions suggest:

```html
<p>
  <span>
    <span class="bold">Total Fat</span> 8g <span class="bold">10%</span>
  </span>
</p>
```

Reply:
@ShaunSHamilton Yes, you're right! I have tried to rectify it. Hopefully the wording is more accurate and helpful now. Let me know if there's anything I can do to improve it. Thanks for the feedback! 🙂